src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import java.io.IOException;
+import java.nio.file.Files;
```

Comment:
Ordering of import statements is inconsistent. Try to order it according to alphabetical order and segregate the types of imports as well for clarity.